sdk/core/System.ClientModel/src/Convenience/AsyncCollectionResultOfT.cs

```diff
@@ -4,41 +4,38 @@
 using System.ClientModel.Primitives;
 using System.Collections.Generic;
 using System.Threading;
+using System.Threading.Tasks;
 
 namespace System.ClientModel;
 
+#pragma warning disable CS1591 // public XML comments
```

Comment:
Adding a note so that we don't lose track of this.

Reply:
Added these.